block-producer/src/lib.rs

```diff
@@ -8,13 +8,15 @@ use tokio::sync::RwLock;
 pub mod test_utils;
 
 pub mod batch_builder;
+pub mod block;
 pub mod block_builder;
+pub mod cli;
+pub mod config;
+pub mod server;
```

Comment:
These 3 modules are not included in the PR; did you forget to push them somehow?

Reply:
ah, oops, I did a `git add -p` and not a `git add`, the new files were no included. fixed!